Code/Tools/LuaIDE/Source/LUA/LUADebuggerComponent.cpp

```diff
@@ -69,26 +72,193 @@ namespace LUADebugger
 
     void Component::Activate()
     {
+        m_remoteTools = AzFramework::RemoteToolsInterface::Get();
         LUAEditorDebuggerMessages::Bus::Handler::BusConnect();
-        AzFramework::TargetManagerClient::Bus::Handler::BusConnect();
-        AzFramework::TmMsgBus::Handler::BusConnect(AZ_CRC("ScriptDebugger", 0xf8ab685e));
+        AZ::SystemTickBus::Handler::BusConnect();
     }
 
     void Component::Deactivate()
     {
+        AZ::SystemTickBus::Handler::BusDisconnect();
         LUAEditorDebuggerMessages::Bus::Handler::BusDisconnect();
-        AzFramework::TargetManagerClient::Bus::Handler::BusDisconnect();
-        AzFramework::TmMsgBus::Handler::BusDisconnect(AZ_CRC("ScriptDebugger", 0xf8ab685e));
+        m_remoteTools = nullptr;
+    }
+
+    void Component::OnSystemTick()
+    {
+        if (m_remoteTools)
+        {
+            const AzFramework::ReceivedRemoteToolsMessages* messages = m_remoteTools->GetReceivedMessages(AzFramework::LuaToolsKey);
+            if (messages)
+            {
+                for (const AzFramework::RemoteToolsMessagePointer& msg : *messages)
+                {
+                    if (AzFramework::ScriptDebugAck* ack = azdynamic_cast<AzFramework::ScriptDebugAck*>(msg.get()))
+                    {
+                        if (ack->m_ackCode == AZ_CRC("Ack", 0x22e4f8b1))
+                        {
+                            if (ack->m_request == AZ_CRC("Continue", 0x13e32adf) || ack->m_request == AZ_CRC("StepIn", 0x761a6b13) ||
+                                ack->m_request == AZ_CRC("StepOut", 0xac19b635) || ack->m_request == AZ_CRC("StepOver", 0x6b89bf41))
+                            {
+                                EBUS_EVENT(LUAEditor::Context_DebuggerManagement::Bus, OnExecutionResumed);
```

Comment:
This can update the EBus calls to not use the Macro, as well as update the AZ_CRC macro to AZ_CRC_CE